src/app/payment/paymentMethod/DigitalRiverPaymentMethod.spec.tsx

```diff
@@ -60,14 +60,34 @@ describe('when using Digital River payment', () => {
 
         expect(component.props())
             .toEqual(expect.objectContaining({
-                containerId: `${method}-component-field`,
+                containerId: `${method.id}-component-field`,
                 initializePayment: expect.any(Function),
                 method,
             }));
     });
 
-    it('initializes method with required config', () => {
-        const container = mount(<PaymentMethodTest { ...defaultProps } method={ method } />);
+    it('initializes method with required config including initializationData', () => {
+        const methodWithInitializationData = {
+            ...method,
+            initializationData: {
+                disabledPaymentMethods: [
```

Comment:
Now this data is coming from the BE, the front-end only knows that this is an array of strings - it doesn't care what those strings are. 

So we can simplify our tests to just have shorter representations of the real data.

I.e. this array could just be something like `['some-method', 'another-method']`